app/controllers/application_controller/tenancy.rb

```diff
@@ -9,11 +9,10 @@ module ApplicationController::Tenancy
   end
 
   def current_tenant
-    # current_user.try(:current_group).try(:tenant) || Tenant.default_tenant
     @current_tenant ||=
      #  Tenant.where(:subdomain => request.subdomains.last).first ||
      #  Tenant.where(:domain => request.domain).first ||
-      Tenant.default_tenant
+      current_user.try(:tenant) || Tenant.default_tenant
```

Comment:
@kbrock maybe we should just change the user.rb from:
`delegate   :current_tenant, :to => :current_group, :allow_nil => true`

to: 
`delegate   :tenant, :to => :current_group, :allow_nil => true`

That way, User#tenant and MiqGroup#tenant are the consistent?